assets/konnectivity/konnectivity-server-deployment.yaml

```diff
@@ -0,0 +1,98 @@
+---
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: konnectivity-server
```

Comment:
I was told that the Konnectivity server can run HA now with 3 replicas.  Is this true?

Reply:
yes it can, but I think we'd like 1 for now until Konnectivity become more stable